CHANGELOG.md

```diff
@@ -145,6 +145,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
     * `WithChainID` / `WithBlockHeight` / `WithBlockHeader` must be used to set values on the context
 * (x/bank) [#17569](https://github.com/cosmos/cosmos-sdk/pull/17569) `BurnCoins` takes an address instead of a module name
 * (x/distribution) [#17670](https://github.com/cosmos/cosmos-sdk/pull/17670) `AllocateTokens` takes `comet.VoteInfos` instead of `[]abci.VoteInfo`
+* (x/authz) [#17741](https://github.com/cosmos/cosmos-sdk/pull/17741) Remove `NewMsgAuthorizeCircuitBreaker`, `NewMsgTripCircuitBreaker`, `NewMsgResetCircuitBreaker` were removed in favor of direct creation of the types where they are used
```

Comment:
I should be circuit instead of authz

Reply:
moved thanks.

Reply:
https://github.com/cosmos/cosmos-sdk/pull/17741#issuecomment-1719663046 😬